bundles/ch.elexis.core.ui/plugin.xml

```diff
@@ -776,6 +776,12 @@
             id="ch.elexis.core.ui.commands.printAddressLabel"
             name="%elexis.printAddresslabel">
       </command>
+      <command
+            categoryId="ch.elexis.core.ui.commands.kategorie"
+            defaultHandler="ch.elexis.core.ui.commands.PrintContactLabelHandler"
+            id="ch.elexis.core.ui.commands.printContactLabel"
+            name="%elexis.printContactLabel">
```

Comment:
Can't find this string in plugin.properties

Reply:
Fixed in a38492ea7a255b9332e9f0f7a5be685064fda861